_data-prepper/pipelines/configuration/processors/otel-metrics-string.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: OpenTelemetry metrics string
```

Comment:
No need for "string" in this text.

Also, the file name has "string" in it. It should probably be `otel-metrics-raw.md`.